src/cf-forms/molecules/big-checkboxes.less

```diff
@@ -0,0 +1,89 @@
+.m-big-checkbox {
+    margin-bottom: unit( 15px / @base-font-size-px, em );
+
+    &:last-child {
+        margin-bottom: unit( 30px / @base-font-size-px, em );
+    }
+    .a-input {
+        @checkbox-width: unit( 18px / @base-font-size-px, em );
+
+        &[type="checkbox"] {
+            .u-visually-hidden();
+
+            .lt-ie9 & {
```

Comment:
properties should be ordered according to front-end standards:

https://github.com/cfpb/front-end/blob/master/css.md#property-order